sdcm/tester.py

```diff
@@ -496,7 +496,8 @@ def get_nemesis_class(self):
             except ValueError:
                 nemesis_name = klass.split(':')[0]
                 num = 1
-            nemesis_threads.append({'nemesis': getattr(nemesis, nemesis_name), 'num_threads': int(num)})
+            nemesis_threads.append({'nemesis': getattr(nemesis, nemesis_name), 'num_threads': int(num),
+                                    'name': nemesis_name})
```

Comment:
where is it used?

Reply:
you can use `__name__` on the class to get its name, why do you need to store it?

Reply:
It was used in a place I no longer using.